lib/bloc/explorer/explorer_bloc.dart

```diff
@@ -156,6 +156,29 @@ class ExplorerBloc extends Bloc<ExplorerEvent, ExplorerState> {
     if (syncWalletSubscription != null) syncWalletSubscription!.cancel();
   }
 
+  Future<void> _syncWalletEvent(
```

Comment:
Could we move this function back below `_syncSingleAccount`?

Reply:
No we can't, `_syncSingleAccount` is used for syncing a specific address and update the balance in receive screen.